### PR TITLE
Adds dental implant surgery

### PR DESCRIPTION
**Pull request by Incoming5643**
While targeting the mouth drill a hole in a tooth then stick a pill in there for hands free later use.

Should hopefully make better friends of chemistry and medbay. Finally found a use for the surgical drill!

### CODE REVIEW

**Comment by Cheridan** [comment]
the dentist job i joked about for so long

it's finally happening

**Comment by Supermichael777** [comment]
GIVE NUKEOPS THEIR CYANIDE PILLS LIKE THIS.

**Comment by Incoming5643** [comment]
Nuke ops have the tools to implant their own pills on the shuttle. I don't know if I should have them start with it pre implanted because they have a "instantly die" button without a confirmation screen right next to "toggle suit slowdown".

**Comment by PKPenguin321** [comment]
nuke ops actually dont get cyanide pills at all anymore since they would use them to kill the crew by stunning -> forcefeeding the pill

**Comment by Fox-McCloud** [comment]
If the nukies started off with the pill pre-implanted, they wouldn't be able to do that anymore.

File: code/modules/surgery/generic_steps.dm [hunk 1]
```diff
@@ -86,4 +86,17 @@
...
 
 	user.visible_message("[user] saws [target]'s [parse_zone(target_zone)] open!", "<span class='notice'>You saw [target]'s [parse_zone(target_zone)] open.</span>")
-	return 1
\ No newline at end of file
+	return 1
+
+//drill bone
+/datum/surgery_step/drill
+	name = "drill bone"
+	implements = list(/obj/item/weapon/surgicaldrill = 100, /obj/item/weapon/pickaxe/drill = 60, /obj/item/mecha_parts/mecha_equipment/drill = 60, /obj/item/weapon/screwdriver = 20)
```
Comment (Fox-McCloud) [inline code/modules/surgery/generic_steps.dm]: I'm pretty sure a mining drill or a GIGANTIC mech drill wouldn't even remotely fit into someone's mouth. Pretty amusing though.

**Comment by PKPenguin321** [comment]
jesus christ getting dental done with a screwdriver would be a horrible experience

File: code/modules/surgery/generic_steps.dm (hunk 1 above)
Comment (Anonus) [inline code/modules/surgery/generic_steps.dm]: If you fucking get a mecha drill and use it for this, you deserve to have the ability.

**Pull request merged**